images/fluent-bit/tests/main.tf

```diff
@@ -27,6 +27,8 @@ resource "helm_release" "fluent-bit" {
       repository = data.oci_string.ref.registry_repo
       tag        = data.oci_string.ref.pseudo_tag
     }
+
+    command = ["/usr/bin/fluent-bit"]
```

Comment:
Is this needed?  It’s already the entrypoint

Reply:
so the helm chart rewrites the entrypoint to `/fluent-bit/bin/fluent-bit` so we need to explicitly set it to the path in our image

Reply:
Can you just throw that in a comment for now, so it's clear later?  I also started a meta-thread on how we document the knobs we need for Helm compat for users (vs. just our tests!)

Reply:
yep, added instructions in the README for using the helm chart as well.